ci/tests/test_01_index_ci/test_01_malformed_index/test_index/test_04/test1.yml

```diff
@@ -0,0 +1,10 @@
+Projects:
+  - id: 1
+    app-id: test1
+    job-id: project1
+    git-path: projects/baseproject/project1
```

Comment:
Yes, each test_xx corresponds to a test_xx_do_something function in the __init__ in same tests caegory directory. This one corresponds to https://github.com/CentOS/container-pipeline-service/pull/129/files#diff-d6f32f3093ffdb1abe1da59abed035eaR27 which checks for failure of index ci if git url is missing in index